### DISCUSSION TITLE
Appium inspector not launching my app

### DISCUSSION THREAD

**Discussion by divi243** [appium/appium#16297] [Q&A] [answered]
Hi Team,

i have used capability like below
{
  "platformName": "iOS",
  "appium:platformVersion": "15.0",
  "appium:deviceName": "(iPhone)(((?!(?i)SE).)*$)",
  "appium:deviceOrientation": "portrait",
  "appium:app": "storage:filename=xyzappname.ipa",
  "appium:appiumVersion": "1.22.0",
  "appium:autoAcceptAlerts": true
}

and i am trying to connect to sauce labs ios real device.
earlier with same capability inspector will launch the device and start the app where i can directly inspect but now i see appium.io download page there is no error but it is also not starting my app.

<img width="1301" alt="Screenshot 2022-01-04 at 3 28 30 PM" src="https://user-images.githubusercontent.com/24429538/148055462-e9884239-f241-4677-a8c3-ba21ce940308.png">

is there something i am missing? i tried to check online but looks like my capability is fine. i am not sure what is going wrong here. please help me in this, thanks in advance!

Regards,
Divita

**Comment by KazuCocoa** [comment, marked as answer]
I think you should ask their support team rather than here since it looks like their environment-specific issue.

**Reply by smithabachu** [reply]
Facing same issue on appium desktop version 1.22 for mac- Inspector not lauching on any ios devices

Failed to create session. An unknown server-side error occurred while processing the command. Original error: The desired capabilities must include either an app or a bundleId for iOS - error is coming

**Comment by divi243** [comment]
thanks for responding but if you mean to say sauce labs issue but it works from our automation and only happening with this version of appium inspector tool. 1.22. 

when i send same capabilities to same device from my automation it works installs the app but i can not locate elements so i am trying to use the inspector tool

Thanks,
Divita

**Reply by KazuCocoa** [reply]
I guessed it was how Sauce Lab handled `"appium:app": "storage:filename=xyzappname.ipa",`. I cannot say anything without Appium server log since such logs have the exact capabilities appium receives.

So far, one possible case is your automation created a session with W3C spec capabilities, while your inspector (I assume you meant https://github.com/appium/appium-desktop, not new https://github.com/appium/appium-inspector) created a session as non-W3C. Then, their environment could not handle W3C capabilities (`app`) well since the desktop sent non-W3C, so their appium started their default configuration.

Anyway, I don't know of how Sauce Labs handles appium capabilities in their product. So it would be nice to ask them first.

**Comment by divi243** [comment]
ok thank you, i will check with them.

**Reply by smithabachu** [reply]
hi divi , iam unable to find webdriver agent project in iOS , i have installed appium 1.22 version on mac big sur , can you please guide, where i can find
